lib/sinon/util/core/every.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+// This is an `every` implementation that works for all iterables
+module.exports = function (obj, fn) {
```

Comment:
Name anonymous function to aid debugging (ref [previous review](https://github.com/sinonjs/sinon/pull/1215#discussion_r94131858)).